implementations/rust/ockam/ockam_api/src/authenticator/enrollment_tokens/acceptor_client.rs

```diff
@@ -15,6 +15,7 @@ pub trait TokenAcceptor {
 
 #[async_trait]
 impl TokenAcceptor for AuthorityNodeClient {
+    #[instrument(skip_all, fields(token = token.to_string()))]
     async fn present_token(&self, ctx: &Context, token: OneTimeCode) -> miette::Result<()> {
```

Comment:
Token is a sensitive field, maybe we can track the hash instead?

Reply:
Done.